### DISCUSSION TITLE
Another thank you!!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [smartcontractkit/full-blockchain-solidity-course-py#1427] [General]
To everyone who has been helping keep this community alive... a massive thank you!

I can see you all engaging and learning so much, and I wanted to give another massive thank you to everyone learning and growing here :)

As always, a major shout out to @cromewar  for all his hard work here!!

**Comment by cromewar** [comment]
Thanks to you, this course is amazing and the community is really helpful, hope to see more amazing people on the hardhat version ;)
